go/otgrpc/options.go

```diff
@@ -17,6 +17,21 @@ func LogPayloads() Option {
 	}
 }
 
+// SpanFilterFunc provides an optional mechanism to decided whether or not
+// to trace a given gRPC call. Return true to create a Span and initiate
+// tracing, false to not create a Span and not trace
+type SpanFilterFunc func(
```

Comment:
sorry, one last nitpick: let's call it `IncludeSpanFunc`... some people think "filter" means keep, others think "filter" means discard.